naga/src/front/wgsl/parse/ast.rs

```diff
@@ -284,7 +284,7 @@ pub enum StatementKind<'a> {
     },
     Increment(Handle<Expression<'a>>),
     Decrement(Handle<Expression<'a>>),
-    Ignore(Handle<Expression<'a>>),
+    Phony(Handle<Expression<'a>>, Span),
```

Comment:
The rename is good: we should hew to the spec's terminology as much as possible.

Why does this need a Span when no other variant of `StatementKind` does? Why isn't the `Span` in `struct Statement` good enough?

Reply:
This span refers to `_` and is used when adding named expressions where we usually add span of var name, but it would make sense for phony statements to use whole statement span (I didn't know we got it).